lib/dpp/isolation/dpp/bootstrapIsolateFactory.js

```diff
@@ -37,14 +37,34 @@ function bootstrapIsolateFactory(dpp, snapshot, isolateOptions) {
       // and do not user bundles for browsers
       global.location = { href: 'shim' };
 
+      const { DashPlatformProtocol, ExternalDataProvider, createTimeoutShim } = DPP;
+
       // Isolates do not have timers, so we need to add them
       const timeoutFunction = createTimeoutShim();
       global.setImmediate = timeoutFunction;
       global.setTimeout = timeoutFunction;
-      const dataProviderWrapper = createDataProvider();
-      global.dataProviderWrapper = dataProviderWrapper;
-      global.dpp = createDpp(dataProviderWrapper);
-      `);
+
+      const dataProvider = new ExternalDataProvider(dataProviderExternalReference)
+      global.dataProviderWrapper = dataProvider;
+
+      global.dpp = new DashPlatformProtocol({ dataProvider });
+    `);
+
+    // await context.eval(`
```

Comment:
Do you think we still will need this in the future? It still stored in the git history